templates/addons/s3/outputs.yml

```diff
@@ -0,0 +1,6 @@
+{{.BucketResourceName}}BucketName:
```

Comment:
You might have to use the sanitizer function for this. CF names are very particular.